helm/nginx-ingress-controller-app/templates/rbac.yaml

```diff
@@ -1,47 +1,41 @@
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: {{ .Values.controller.name }}
+  name: {{ include "resource.default.name" . }}
   labels:
-    app: {{ .Values.controller.name }}
-    giantswarm.io/service-type: "managed"
-    app.kubernetes.io/name: {{ .Values.controller.k8sAppLabel }}
+    {{- include "labels.common" . | nindent 4 }}
 subjects:
 - kind: ServiceAccount
-  name: {{ .Values.controller.name }}
+  name: {{ include "resource.default.name" . }}
   namespace: {{ .Release.Namespace }}
 roleRef:
   kind: ClusterRole
-  name: {{ .Values.controller.name }}
+  name: {{ include "resource.default.name" . }}
   apiGroup: rbac.authorization.k8s.io
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: {{ .Values.controller.name }}
+  name: {{ include "resource.default.name" . }}-ng
```

Comment:
Old Role name was configurable and defaulted to `nginx-ingress-role`. Now Role name is not configurable, it's determined by Helm Release name, to be unique across multiple releases.
 
Role ref in RoleBinding is immutable. As workaround I resorted to adding a suffix ("-ng" for next generation) to force RoleBinding to be recreated; old one gets to be cleaned up as part of Helm release upgrade.